texar/modules/classifiers/bert_classifiers.py

```diff
@@ -0,0 +1,284 @@
+                cache_dir=cache_dir,
+                hparams=encoder_hparams)
+
+            # Creates an dropout layer
+            drop_kwargs = {"rate": self._hparams.dropout}
+            layer_hparams = {"type": "Dropout", "kwargs": drop_kwargs}
+            self._dropout_layer = layers.get_layer(hparams=layer_hparams)
+
+            # Creates an additional classification layer if needed
+            self._num_classes = self._hparams.num_classes
+            if self._num_classes <= 0:
+                self._logit_layer = None
+            else:
+                logit_kwargs = self._hparams.logit_layer_kwargs
+                if logit_kwargs is None:
+                    logit_kwargs = {}
+                elif not isinstance(logit_kwargs, HParams):
+                    raise ValueError(
+                        "hparams['logit_layer_kwargs'] must be a dict.")
+                else:
+                    logit_kwargs = logit_kwargs.todict()
+                logit_kwargs.update({"units": self._num_classes})
+                if 'name' not in logit_kwargs:
+                    logit_kwargs['name'] = "logit_layer"
+
+                layer_hparams = {"type": "Dense", "kwargs": logit_kwargs}
+                self._logit_layer = layers.get_layer(hparams=layer_hparams)
+
+    @staticmethod
+    def default_hparams():
+        """Returns a dictionary of hyperparameters with default values.
+
+        .. code-block:: python
+            {
+                # (1) Same hyperparameters as in BertEncoder
+                ...
+                # (2) Additional hyperparameters
+                "num_classes": 2,
+                "logit_layer_kwargs": None,
+                "clas_strategy": "cls_time",
+                "max_seq_length": None,
+                "dropout": 0.1,
+                "name": "bert_classifier"
+            }
+
+        Here:
+
+        1. Same hyperparameters as in
+        :class:`~texar.modules.BertEncoder`.
+        See the :meth:`~texar.modules.BertEncoder.default_hparams`.
+        An instance of BertEncoder is created for feature extraction.
+
+        2. Additional hyperparameters:
+
+            "num_classes" : int
+                Number of classes:
+
+                - If **`> 0`**, an additional :tf_main:`Dense <layers/Dense>` \
+                layer is appended to the encoder to compute the logits over \
+                classes.
+                - If **`<= 0`**, no dense layer is appended. The number of \
+                classes is assumed to be the final dense layer size of the \
+                encoder.
+
+            "logit_layer_kwargs" : dict
+                Keyword arguments for the logit Dense layer constructor,
+                except for argument "units" which is set to "num_classes".
+                Ignored if no extra logit layer is appended.
+
+            "clas_strategy" : str
+                The classification strategy, one of:
+                - **"cls_time"**: Sequence-leve classification based on the \
```

Comment:
typo `Sequence-leve` --> `Sequence-level`

Reply:
To make it clearer: `output of the first time step (CLS)` --> `output of the first time step (which is the "CLS" token)`